lib/osutil/atomic.go

```diff
@@ -105,6 +110,12 @@ func (w *AtomicWriter) Close() error {
 		w.err = err
 		return err
 	}
+	if info != nil {
```

Comment:
Info might theoretically be non-nil yet garbage if the file didn't exist. (As in I think info is undefined if err != nil)